jobs/build/publish-rpms/Jenkinsfile

```
@@ -0,0 +1,47 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    buildlib = load( "pipeline-scripts/buildlib.groovy" )
+    commonlib = buildlib.commonlib
+    commonlib.describeJob("publish-rpms", """
+        <h2>Automatically publishing rpms to mirror</h2>
+    """)
+
+    properties(
+        [
+            disableResume(),
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: ''
+                )
+            ),
+            [
+                $class : 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    string(
+                        name: 'BUILD_VERSION',
+                        description: 'Target release version',
+                        trim: true,
+                    ),
+                ]
+            ],
+            disableResume(),
+            disableConcurrentBuilds()
+        ]
+    )
+    commonlib.checkMock()
+
+    version = params.BUILD_VERSION
+    scripts_dir = "${env.WORKSPACE}/hacks/rhel-7-deps-mirroring/collect-deps.sh"
```

shouldn't this be... the directory?
```suggestion
    scripts_dir = "${env.WORKSPACE}/hacks/rhel-7-deps-mirroring/"
```
also i would rather the script be under the job, not sourced from somewhere we might change without realizing it's used here.